src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
+    static String sectionDivider = "____________________________________________________________";
+    public static void greetUser(){
+        String greeting = "\t" + sectionDivider + "\n"
+                + "\tHello! I'm Duke. \n"
+                + "\tWhat can I do for you? \n"
+                + "\t" + sectionDivider;
+        System.out.println(greeting);
+    }
+    public static void signOff(){
+        String sign_off = "\t" + sectionDivider + "\n"
+                + "\tBye. Hope to see you again soon!\n"
+                + "\t" + sectionDivider;
+        System.out.println(sign_off);
+    }
+
+    static int numberOfTasks = 0;
+    static Task tasks[] = new Task[100];
```

Comment:
Well done! You could consider using a named constant for 100, say "MAX_TASKS_SIZE".